package/src/monaco.contribution.ts

```diff
@@ -87,6 +95,7 @@ export function setupMonacoKusto(monacoInstance: typeof monaco) {
         return {
             kustoDefaults,
             getKustoWorker,
+            getKustoWorkerTokenParsing,
```

Comment:
Any reason that it's public?

Reply:
Yes, because sometimes we want to use a method that relies o nthe token parsing, for example: "getClusterReferences".
We want to use it  in the token parsing worker, and not get the default one to get stuck

Reply:
is it easy to identify all the interfaces that need the token parser? 
it would be nice if the route will be done internally?

Reply:
It cannot be done, it's a method on the worker itself

Reply:
It's not easy to identify + if someone add such a method, he will probably wouldn't know he needs to route it internally between the two workers..